docs/development/setup.md

```diff
@@ -34,6 +42,9 @@ the manifests is available via
 make kind-install
 ```
 
+**Note**: If a `kind context` different to the default is used, be aware that the `Makefile` is referencing the default
```

Comment:
I would note here, that the user should make sure the `kind get clusters` is pointing to a `kind` cluster in case he has multiple environments running.